ios/fastlane/Fastfile

```diff
@@ -101,10 +101,8 @@ platform :ios do
       api_key: api_key,
       skip_waiting_for_build_processing: true,
       distribute_external: true,
-      changelog: changelog
+      changelog: changelog,
+      ipa: '../build/ios/ipa/Memo.ipa'
     )
-
-    # Clean local dSYM and IPA files
-    clean_build_artifacts
```

Comment:
We don't need to clean artifacts anymore because `flutter build ipa` builds to a temporary directory, so the archive and the DSyms are not placed in the root directory anymore